dev/io.openliberty.microprofile.metrics.internal.3.0.monitor/src/io/openliberty/microprofile/metrics/internal/monitor/MetricsJaxRsEMCallbackImpl.java

```diff
@@ -117,8 +125,8 @@ public void getSharedMetricRegistries(SharedMetricRegistries sharedMetricRegistr
 
 		if (fullMethodSignature == null || fullyQualifiedClassName == null || fullMethodSignature.isEmpty()
 				|| fullyQualifiedClassName.isEmpty()) {
-			throw new IllegalStateException("The following values are either null or empty - fullyQualifiedClassName: "
-					+ fullyQualifiedClassName + " fullMethodSignature: " + fullMethodSignature);
+		    Tr.warning(tc, "Could not resolve the class name or method signature class:[" + fullMethodSignature + "] method:[" +  fullMethodSignature+"]");
```

Comment:
The warning message needs to be translated.

Reply:
Since an empty `ResourceInfo` with no resource class or resource method originates from a non-existing URI, change to be a debug statement.

Reply:
The new commit also formats the class, only change is to the `Tr.warning` to `Tr.debug`

Reply:
Did you push your changes?